README.md

```diff
@@ -27,7 +27,11 @@ upper_case_full_names = []
 
 ### Answer
 
-```rb
+upper_case_full_names = [students[0][:first_name].upcase + " "+ students[0][:last_name].upcase , students[1][:first_name].upcase + " "+ students[1][:last_name].upcase , students[2][:first_name].upcase + " "+ students[2][:last_name].upcase]
+
+  
+p upper_case_full_names
```

Comment:
Try to create a program that would work with any number of elements in your students' array, don't hardcode your functionality. Consider looping through the array